README.md

```diff
@@ -192,21 +192,20 @@ please join the [**#v8go**](https://gophers.slack.com/channels/v8go) channel on
 
 ### Windows
 
-In order to build a project using v8go on Windows, Go requires a gcc compiler to be installed.
+There used to be Windows binary support. For further information see, [this PR](UPDATE).
```

Comment:
This can be filled in now that the PR is open.

```suggestion
There used to be Windows binary support. For further information see, [PR #234](https://github.com/rogchap/v8go/pull/234).
```

Reply:
Thank you!